### DISCUSSION TITLE
Is there a way to control flet's card shadow in order to obtain Nuemorphic design ?

### DISCUSSION THREAD

**Discussion by Haadi-A** [flet-dev/flet#970] [Q&A]
@ndonkoHenri , @DavMelchi have designed my landing page and I want to implement Nuemorphic design to my buttons. But all that I know I can try is to use flet's card and set the elevation in such a way that there is some shadow beneath it but that only creates the illusion of some sort of elevation. But I don't want to just create elevation, I want more control over the card because to do Nuemorphic design you have to set the shadow both under the card and on one side to simulate light source but the flet card only allows you to control the elevation, although flet's default shadow setting ends up looking awesome, I need more control to in order to do stuff like this:

 
![Nuemorphic design](https://user-images.githubusercontent.com/77079200/216663955-3f72fc26-225f-42c6-9479-de6e1a9fb63c.png)

So is there a way to do this in flet?

**Comment by ndonkoHenri** [comment]
https://github.com/flet-dev/flet/issues/880
There is an issue to help achieve that.

**Comment by FeodorFitsner** [comment]
Look at https://github.com/flet-dev/flet/issues/348 please - is that kind of shadow that will help to implement neumorphic design?

**Reply by Haadi-A** [reply]
Exactly if all those BoxShadow knobs are implemented in flet (ie.Being able to set the shadows x and y offsets, spread size, spread radius and shadow spread color) and this can be applied to Containers, Buttons and Cards, then neumorphic designs would absolutely be possible.

**Reply by FeodorFitsner** [reply]
What about radios, checkboxes?

**Reply by Haadi-A** [reply]
Sure, if the shadow feature can also be implemented to it then I think we should go for it, cuz normally neumorphic design is applied to buttons and containers. But if they can be applied to radio buttons and checkboxes then I think it will be great cuz even right now I can see how you could apply the neumorphic design to radios and check buttons.

**Reply by ndonkoHenri** [reply]
BoxShadow. 🤔 
[Flutils](https://flutils.pages.dev) will definitely have a tab dedicated to that. 🙂

**Reply by FeodorFitsner** [reply]
I don't see a way to control shadow in [`Card` widget](https://api.flutter.dev/flutter/material/Card-class.html), but `Container` has that ability for sure.
